schema.sql

```diff
@@ -8,3 +8,24 @@
     neutered boolean,
     weight_kg integer
 );
+
+ALTER TABLE animals
+ADD COLUMN species_id INT,
+ADD COLUMN owner_id INT;
+
+CREATE TABLE owners (
+    id  INT generated by default as identity primary key, 
+    full_name VARCHAR(100), 
+    age INT
+);
+
+CREATE TABLE species (
+    id  INT generated by default as identity primary key,
+    name VARCHAR(100)
+);
+
+INSERT INTO species (name)
+VALUES ('Pokemon');
+
+INSERT INTO species (name)
+VALUES ('Digimon');
```

Comment:
- [ ] Good job so far, implementing the changes requested by the previous reviewer, :clap: However, it will be really nice if you put your INSERT QUERIES in the `data.sql` file according to the project requirement which says: `Everything that populates tables with data should be saved in the data.sql file.`